### PR TITLE
feature done

### PR DESCRIPTION
**Pull request by Dash293**

### CODE REVIEW

**Review by ThomasDCW — APPROVED** [review]
ok

**Review by ThomasDCW — DISMISSED** [review]
okokok

File: backend/src/router.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ router.get("/users", UserController.browse);
 router.get("/users/:id", UserController.read);
 router.put("/users/:id", UserController.edit);
 router.get("/topics", TopicController.browse);
+router.get("/!empty-topics", TopicHaveLessonController.browse);
```
Comment (LuigiMenez) [inline backend/src/router.js:39]: Not " ! " in the route

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Not " ! " in the route

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**